src/platform/assets/components/AssetGrid.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div
+    data-component-id="AssetGrid"
+    class="grid grid-cols-2 md:grid-cols-4 gap-6 p-8"
+    role="grid"
+    aria-label="Asset collection"
+    :aria-rowcount="-1"
+    :aria-colcount="-1"
+    :aria-setsize="assets.length"
+  >
+    <AssetCard
+      v-for="(asset, index) in assets"
+      :key="asset.id"
+      :asset="asset"
+      role="gridcell"
+      :aria-posinset="index + 1"
```

Comment:
It's weird to see this without also seeing omission behavior.
Normally I'd expect the for to loop over a partial view of the total set and the position to be the index within the collection as a whole.